schemas/iso19139/src/main/plugin/iso19139/layout/config-editor.xml

```diff
@@ -2137,6 +2159,17 @@
 
     </view>
     <view name="advanced">
+      <sidePanel>
```

Comment:
An improvement could be to allow also `sidePanel` at the `views` level so applies to any view not having an explicit `sidePanel` to simplify the configuration. If a view doesn't require `sidePanel` could add an empty `<sidePanel />`.

It's not something really mandatory as usually only 2-3 views and also the snippet is quite small.